libraries/scopes/src/simplify.js

```diff
@@ -0,0 +1,22 @@
+exports.simplifyScopeExpression = (scope) => {
+  let scopeSet = [];
+  scopeSet.push(scope);
+  removeDuplicateScopes(scope);
+  console.log(scopeSet[0]);
+  return scopeSet[0];
+};
+
+function removeDuplicateScopes(scope){
+  if(Object.keys(scope).includes('AnyOf') || Object.keys(scope).includes('AllOf')){
+    const values = (Object.values(scope)[0]);
+    if(ifDuplicateExists(values)){
```

Comment:
I like the approach of `[...new Set(values)]` to remove duplicates!  However, given a list without duplicates that will just reproduce a list with the same items, so there's no need to test for duplicates beforehand -- just always use the approach.

I suspect you added this because sometimes the list of scopes came out in a different order?  For consistency, you could sort the scopes -- that's a pretty common tactic in TDD.

The normalizeScopeSet function will both sort and remove duplicates.. I'm curious what didn't work with those?

Reply:
Yes true. When I added this because the `normalizeScopeSet ` function did not work because I didn't add the sort function. Now its working. The only problem is I have a nested array.

![newone](https://user-images.githubusercontent.com/30138596/68646793-28e07300-0542-11ea-912b-ce07951a6db9.JPG)

Reply:
Plus, netlify tests are failing because of the following error:

> Module not found: Error: Can't resolve 'taskcluster-lib-scopes' in '/opt/build/repo/libraries/scopes/src'_